src/moneypenny/moneypenny.py

```diff
@@ -122,16 +130,51 @@ async def execute_order(self, verb: str, dossier: Dict[str,Any]) -> None:
         dossier: Dossier associated with the order for the user.
 
         """
-        verb=verb.strip().lower()
+        verb = verb.strip().lower()
         # We re-read the orders each time; the config map is allowed to change
         await self._read_orders()
```

Comment:
If you're going to re-read this all the time, instead of using a member variable which implies caching, why not just have this return the thing stored in self.orders and get rid of that entirely?  Then it's obvious you aren't using the cached one.  For example, you are also setting this in init and I don't know if that's used before it is reread?

Reply:
I got rid of self.orders and self.dossier entirely.